Makefile

```diff
@@ -7,7 +7,7 @@ SHAREDIR ?= ${PREFIX}/share
 PYTHON ?= $(shell command -v python3 python|head -n1)
 DESTDIR ?= /
 PATH := $(PATH):$(HOME)/.local/bin
-MYPIP ?= pip
+MYPIP ?= uv pip
```

Comment:
This failed CI https://konflux-ui.apps.kflux-prd-rh03.nnv1.p1.openshiftapps.com/ns/ramalama-tenant/applications/ramalama/pipelineruns/bats-on-pull-request-hmc9w/logs?task=build-images
` /bin/sh: line 1: uv: command not found`